tests/neg-macros/annot-codegen/Macro_1.scala

```diff
@@ -19,24 +19,23 @@ class data extends MacroAnnotation:
           val withParam = With(param)
           val paramType = cls.declaredField(param).info
           val existingOpt =
-            cls.declaredMethod(withParam).find(o =>
-              val paramss = o.paramSymss
-              paramss.size == 1 && paramss(0).size == 1
+            cdef.body.find(stat =>
+              val paramss = stat.symbol.paramSymss
+              stat.symbol.name == withParam
+              && paramss.size == 1 && paramss(0).size == 1
```

Comment:
@nicolasstucki I understand that we can look into the input tree directly, but in my opinion we should also be able to use Symbol#tree for this usecase, unless there is something that prevents us from using it?